sdk/include/ndk/pstypes.h

```diff
@@ -991,6 +991,17 @@ typedef struct _EPROCESS_QUOTA_BLOCK
     ULONG ProcessCount;
 } EPROCESS_QUOTA_BLOCK, *PEPROCESS_QUOTA_BLOCK;
 
+//
+// Quota Structures
+//
+typedef enum _PS_QUOTA_TYPE
+{
+    PsNonPagedPool,
+    PsPagedPool,
+    PsPageFile,
+    PsQuotaTypes
+} PS_QUOTA_TYPE;
```

Comment:
See https://www.geoffchappell.com/studies/windows/km/ntoskrnl/api/ps/psquota/type.htm
You may want to support the two other values too... (At least, a comment?)